### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/ACS/acs_pixel_area_maps/acs_pixel_area_maps.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#562]
## What
An accessibility scan flagged 6 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/ACS/acs_pixel_area_maps/acs_pixel_area_maps.html because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

The following elements need attention:

- `<pre id="codecell1"><span></span>The following task in the stsci.skypac package can be run with TEAL:
                                    skymatch                                    
</pre>` (selector: `#codecell1`)
- `<pre id="codecell3"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j6me13qhq_flt.fits to ./mastDownload/HST/j6me13qhq/j6me13qhq_flt.fits ...
</pre>` (selector: `#codecell3`)
- `<table class="table">` (selector: `#flt-files-wfc-specific > .pst-scrollable-table-container > table`)
- `<pre id="codecell8">` (selector: `#codecell8`)
- `<pre id="codecell11">` (selector: `#codecell11`)
- `<pre id="codecell12">` (selector: `#codecell12`)

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
